Lib/importlib/_bootstrap_external.py

```diff
@@ -211,7 +212,7 @@ def _write_atomic(path, data, mode=0o666):
         with _io.FileIO(fd, 'wb') as file:
             file.write(data)
         _os.replace(path_tmp, path)
-    except OSError:
+    except:
```

Comment:
`except Exception` at least, so that we don't swallow interrupts.

Reply:
They are not swallowed, there is a `raise` in the handler body.

The bare `except` is needed here because the temporary path should be removed at any error.